backend/src/rest-api-docs.yaml

```diff
@@ -349,28 +367,14 @@ components:
         - tournamentId
         - firstMatchSideId
         - secondMatchSideId
-    PostMatchResponseBody:
```

Comment:
@nowakprojects tutaj w którymś wcześniejszym PR mówiłeś, żeby to usunąć bo było niepotrzebne, a jakoś znów to przemyciłem na developa (chyba dwa PRy temu), dlatego jest już naprawdę usunięte w tym PR